config_src/nuopc_driver/mom_cap.F90

```diff
@@ -1585,6 +1586,16 @@ subroutine ModelAdvance(gcomp, rc)
 
         ! write restart file(s)
         call ocean_model_restart(ocean_state, restartname=restartname)
+
```

Comment:
You need an optional control here. What if we're not running ufs with the och stoch turned on?

Reply:
The ocean_model_restart subroutine immediately returns if there are no stochastic physics active.  The coupled model and data atmosphere regression tests pass.

Reply:
Won't we need new parameters in the MOM_input files to turn stoch on/off? I don't see any param file changes in your ufs-weather-model PR.